src/swell/test/code_tests/slurm_test.py

```diff
@@ -22,7 +22,7 @@ class SLURMConfigTest(unittest.TestCase):
     # configuration and platform-specific settings
     @patch("swell.utilities.slurm.slurm_global_defaults")
     @patch("platform.platform")
-    def test_slurm_config(self, platform_mocked, mock_global_defaults):
+    def test_slurm_config(self, platform_mocked: dict, mock_global_defaults: dict) -> None:
```

Comment:
Is `dict` the right annotation here? I think what these should really be is probably `unittest.mock.Mock`:

```suggestion
    def test_slurm_config(self, platform_mocked: Mock, mock_global_defaults: Mock) -> None:
```

Note that this requires `from unittest.mock import Mock` above. Or, you can use a forward reference -- `: "Mock"`.

Reply:
I've added a direct hint to Mock